### PR TITLE
Config: adjust a paragraph in 'latex_table_style' documentation

### PR DESCRIPTION
**Pull request by jayaddison**
### Feature or Bugfix
- Refactoring

### Purpose
- Nitpicky rephrasing of a `latex_table_style` documentation paragraph based on something I experienced here: https://github.com/sphinx-doc/sphinx/pull/12986#issuecomment-2400616466

### Detail
- Perhaps the result is too wordy?  (repetition of the word 'table', for example)

### Relates
- N/A

### CODE REVIEW

File: doc/usage/configuration.rst [hunk 1]
```diff
@@ -3046,8 +3046,11 @@ These options influence LaTeX output.
          Please update your project to use the
          :ref:`latex table color configuration <tablecolors>` keys instead.
 
-   Each table can override the global style via ``:class:`` option,
-   or ``.. rst-class::`` for no-directive tables (cf.  :ref:`table-directives`).
+   To customize the styles on an individual table, either set the ``:class:``
+   option when the table's contents are documented inside a table directive,
+   or, otherwise, add a preceding :ref:`rst-class <rstclass>` directive
+   (cf. :ref:`table-directives`).
```
Comment (jayaddison) [inline doc/usage/configuration.rst]:
```suggestion
   To customize the styles on an individual table, either set the ``:class:``
   option on the enclosing table directive - if there is one - or insert a
   :ref:`rst-class <rstclass>` directive before the table otherwise
   (cf. :ref:`table-directives`).
```

File: doc/usage/configuration.rst [hunk 2]
```diff
@@ -3046,8 +3046,11 @@ These options influence LaTeX output.
          Please update your project to use the
          :ref:`latex table color configuration <tablecolors>` keys instead.
 
-   Each table can override the global style via ``:class:`` option,
-   or ``.. rst-class::`` for no-directive tables (cf.  :ref:`table-directives`).
+   To customize the styles on an individual table, either set the ``:class:``
+   option on the enclosing table directive - if there is one - or insert a
+   :ref:`rst-class <rstclass>` directive before the table otherwise
+   (cf. :ref:`table-directives`).
```
Comment (AA-Turner) [inline doc/usage/configuration.rst]: Perhaps?:
```suggestion
   To customise the styles for a table,
   use the ``:class:`` option if the table is defined using a directive,
   or otherwise insert a :ref:`rst-class <rstclass>` directive before the table
   (cf. :ref:`table-directives`).
```

Comment (jayaddison) [reply doc/usage/configuration.rst]: Mostly looks good - is the word `defined` suitable here, though?  (as a reader, it makes sense, but I'm not sure it's consistent with the existing documentation)

Comment (AA-Turner) [reply doc/usage/configuration.rst]: Happy to go with what you think best.

Comment (jayaddison) [reply doc/usage/configuration.rst]: Let's go with that - I think I was being overly-fussy about the word `defined`.

**Comment by jayaddison** [comment]
Thanks!
